rclcpp/include/rclcpp/logger.hpp

```diff
@@ -122,6 +123,7 @@ class Logger
   : name_(new std::string(name)) {}
 
   std::shared_ptr<const std::string> name_;
+  std::shared_ptr<std::pair<std::string, std::string>> logger_sublogger_pairname_ = nullptr;
```

Comment:
Do we even need to store this pair?  That is, we already have the `name_` above, so wouldn't it be sufficient to just store the `suffix` here?

Reply:
Yes, we need it because the `name_` in the Logger is a full name.
I think that it's better to keep the same parameters for `rcl_logging_rosout_remove_sublogger` and `rcl_logging_rosout_add_sublogger`.

Reply:
Oh, I see.  I misunderstood how this works.  OK, now I understand, that makes sense.